dataactbroker/handlers/accountHandler.py

```diff
@@ -21,7 +23,11 @@
 from dataactcore.utils.statusCode import StatusCode
 from dataactcore.interfaces.function_bag import get_email_template, check_correct_password, updateLastLogin
 from dataactcore.config import CONFIG_BROKER
-from dataactcore.models.lookups import USER_STATUS_DICT, PERMISSION_TYPE_DICT, PERMISSION_MAP
+from dataactcore.models.lookups import (
+    PERMISSION_SHORT_DICT, PERMISSION_TYPE_DICT, USER_STATUS_DICT)
```

Comment:
Ok, so it's not like they're doing anything super important that I didn't know about. I was just confused because the function_bag imports aren't changed and they seem longer. I use PyCharm so it just lets me keep going. Doesn't even get to the random vertical line that probably denotes a "page width" of some sort.